bin/dispatch-conf

```diff
@@ -574,8 +574,9 @@ def clear_screen():
     os.system("clear 2>/dev/null")
 
 
-shell = os.environ.get("SHELL")
-if not shell or not os.access(shell, os.EX_OK):
+broot = portage.settings["BROOT"] or "/"
```

Comment:
It's messy for a cli tool to use BROOT outside of the ebuild env like this.

PATH is a fairly standard thing to use here, and we use it ourselves elsewhere in prefix support.

Reply:
How would you feel about using `portage.const.EPREFIX` instead?

I'm basically trying to replicate what libc would do in os.system(). If we don't think that's important I can drop this.

Reply:
I went ahead and updated it to just use PATH instead.

Reply:
Yeah, the glibc patch is reasonable, but here PATH is a reasonable alternative.